manifests/postgresql-operator-default-configuration.yaml

```diff
@@ -61,6 +61,8 @@ configuration:
     # log_s3_bucket: ""
     # kube_iam_role: ""
     aws_region: eu-central-1
+    # additional_secret_mount: 'some-secret-name'
+    # additional_secret_mount_path: '/some/dir'
```

Comment:
does it really work with single quotes? In the ConfigMap I can see other examples with them but not in the `OperatorConfiguration`. Other the other way around: Would `"` work for ConfigMap, too.? If yes, change it also in the `values.yaml` file for Helm chart. The Helm templates for ConfigMap and CRD can stay as they are.

Reply:
using now double quotes